### PR TITLE
Custom Fields Improvements, Graphics Improvements, BugFixes

### PR DESCRIPTION
**Pull request by denysvitali**
- Custom Fields don't require any table customization
- Custom fields are now sorted by order number and label name, take a look at the new custom field form for a better explanation
- Custom fields can now be placed in other groups like "Contact Information", "Taxes Information", "Personal Information"
- Sumex fields are now hidden if the option isn't enabled in settings

@Kovah , this PR doesn't have the migration tool yet.
@Anyone who wants to try this: Upgrading with the included sql file will delete some data from the custom fields, use at your own risk: **don't use it on production data if you don't know what you're doing!**

### CODE REVIEW

**Comment by denysvitali** [comment]
@Kovah , about the "Needs revision" tag: should I do something?

**Comment by Kovah** [comment]
Yes, see the code review

File: assets/core/scss/_core.scss [hunk 1]
```diff
@@ -305,14 +304,17 @@ textarea {
 */
 
 #main-area {
-  display: table-cell;
```
Comment (denysvitali) [inline assets/core/scss/_core.scss]: Why not?
http://caniuse.com/#feat=flexbox

Comment (Kovah) [inline assets/core/scss/_core.scss]: I understand that Caniuse says it's supported but we had several problems even with simple flexbox layouts, especially in Safari both for Mac and iOS. That's why we never really use flexbox for critical elements like the base layout.

Comment (denysvitali) [inline assets/core/scss/_core.scss]: Sorry to bother you, but when was the last time you had problems due to flexbox?

**Comment by denysvitali** [comment]
👍

**Pull request merged**